libsolidity/formal/BMC.h

```diff
@@ -149,7 +149,10 @@ class BMC: public SMTEncoder
 
 		friend bool operator<(BMCVerificationTarget const& _a, BMCVerificationTarget const& _b)
 		{
-			return _a.expression->id() < _b.expression->id();
+			if (_a.expression->id() == _b.expression->id())
```

Comment:
Here's was the problem. We didn't accumulate more than  proved 1 BMC target for a given expression.

Reply:
Right, good catch!